context/app/static/js/pages/LineUpPage/LineUpPage.jsx

```diff
@@ -1,20 +1,52 @@
 import React from 'react';
-import LineUp from 'lineupjsx';
+import LineUp, {
+  LineUpStringColumnDesc,
+  LineUpNumberColumnDesc,
+  LineUpCategoricalColumnDesc,
+  LineUpDateColumnDesc,
+} from 'lineupjsx';
 import 'lineupjsx/build/LineUpJSx.css';
 import Paper from '@material-ui/core/Paper';
 
 import SectionHeader from 'js/shared-styles/sections/SectionHeader';
+import metadataFieldTypes from 'metadata-field-types';
 
 function LineUpPage(props) {
   const { entities } = props;
+  const firstRow = entities[0];
+  const notEnumFields = new Set([
+    'uuid',
+    // Donors:
+    'hubmap_id',
+    'medical_history',
+    // Samples:
+    'donor.hubmap_id',
+    'sample_id',
+    // Datasets:
+    'description',
+    'library_adapter_sequence',
+    'library_id',
```

Comment:
Is there a way to get this from `ingest-validation-tools` or is hard coding this the best route for now?

Reply:
I started off thinking I would want to distinguish the string fields from the enum fields based on their definitions... but it turned out that for most string fields, even if they aren't formally enums, only a small number of values are seen, and having a histogram at the top of the column doesn't seem like a bad thing.

This may be revisited: It wouldn't be that hard to have `field-types.yaml` figure out which fields are enums.